PostFreeSurfer/scripts/CreateMyelinMaps.sh

```diff
@@ -86,21 +98,29 @@ ${CARET7DIR}/wb_command -volume-palette $Jacobian MODE_AUTO_SCALE -interpolate t
 # To detect this situation, and prevent it from occurring, FreeSurferPipeline errors out if the
 # one-step warpfield created here ($OutputOrigT2wToT1w) already exists.]
```

Comment:
Please change this comment to the following to reflect the change to checking $OutputOrigT1wToT1w instead in FreeSurferPipeline.
```
# To detect this situation, and prevent FreeSurferPipeline from running after PostFreeSurferPipeline,
# FreeSurferPipeline errors out if the one-step warpfield created here ($OutputOrigT1wToT1w) already exists.]

```

Reply:
done